frontend/app/src/locales/en.json

```diff
@@ -4307,7 +4307,9 @@
     },
     "hint": {
       "description": "* type the filter and then press enter e.g.",
-      "example": "start = 26/05/2022"
+      "example": "start = 26/05/2022",
+      "edit_note": "Double-click a filter to edit"
```

Comment:
Okay as I said this is not triggered by Double-click, but only single click, so:
```
Click applied filter to edit the value
```